website/content/en/docs/getting-started/_index.md

```diff
@@ -184,10 +184,10 @@ helm repo update
 
 kubectl create namespace monitoring
 
-curl -fsSL https://karpenter.sh/docs/getting-started/prometheus-values.yaml
+curl -fsSL https://karpenter.sh/docs/getting-started/prometheus-values.yaml | tee prometheus-values.yaml
```

Comment:
I'm confused - when I try to curl that path I get a 404. Also is `tee` really needed here? Why isn't curl outputting to the right filename in the first place?

Reply:
The files were removed, I've added them back in this PR. 

Regarding the `tee`, I wanted the user to see what they are configuring rather than blindly passing the values files to the helm command.